projects/plutus/inc/current_calibration.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+// LTC 2484 calibration module
```

Comment:
Please add any modules that are required to be initialized and improve your documentation

Reply:
Fixed

Reply:
my dude